sdks/go/examples/wordcount/wordcount.go

```diff
@@ -137,7 +145,8 @@ var (
 	lineLen         = beam.NewDistribution("extract", "lineLenDistro")
 )
 
-// extractFn is a DoFn that emits the words in a given line and keeps a count for small words.
+// extractFn is a structural DoFn that emits the words in a given line and keeps
+// a count for small words.
```

Comment:
```suggestion
// a count for small words. Its ProcessElement function will be invoked on each
// element in the input PCollection.
```

Optional: This might be helpful context if people aren't familiar with Beam. Not sure if I complied w/ the 80 char limit